src/s2wasm.h

```diff
@@ -159,16 +159,23 @@ class S2WasmBuilder {
     return cashew::IString(str.c_str(), false);
   }
 
-  int32_t getInt() {
-    int32_t ret = 0;
+  uint32_t getInt() {
```

Comment:
Yes; the rule is: overflow on signed types is UB, while overflow on unsigned types is defined silent wraparound.